blueprint/api.apib

```diff
@@ -2209,6 +2268,59 @@ This endpoint allows you to check whether a username is valid (by running a vali
 + data(Stripe Auth Resource)
 + include JSON API Version
 
+## StripeCard (object)
++ data(StripeCard Resource)
++ include JSON API Version
+
+## Stripe Card Attributes (object)
++ brand: (enum[string]) - Card brand.
+    + Members
+        + `Visa`
+        + `American Express`
+        + `MasterCard`
+        + `Discover`
+        + `JCB`
+        + `Diners Club`
+        + `Unknown`
++ cvc_check: (enum[string]) - If a CVC was provided, results of the check.
+    + Members
+        + `pass`
+        + `fail`
+        + `unavailable`
+        + `unchecked`
++ exp_month: 12 (number)
++ exp_year: 2017 (number)
++ id_from_stripe: `card_18Emd52eZvKYlo2C1s9DYdv8` (required, string)
```

Comment:
I actually think these attributes should be dasherized, and surrounded by ` marks.